docs/recipes/complex-journal-entries.md

```diff
@@ -0,0 +1,61 @@
+# Complex (Multi-Debit, Multi-Credit) Journal Entries
```

Comment:
What about "Multi-Debit/Credit Journal Entries"? Or s/journal entries/transfers? (I'm just trying to shorten the title, since it is extracted into the sidebar navigation on https://docs.tigerbeetle.com/!)